development/readme_eclipse_4.32.html

```diff
@@ -176,9 +176,9 @@ <h2 id="TargetOperatingEnvironments2">
   <p>Most of the Eclipse SDK is "pure" Java code and has no direct dependence on the underlying operating system. The chief dependence is therefore on the Java Platform itself. Portions are
     targeted to specific classes of operating environments, requiring their source code to only reference facilities available in particular class libraries (e.g. J2ME Foundation 1.1, J2SE 1.4, Java
     5, etc).</p>
-  <p>In general, the 4.32 release of the Eclipse Project is developed on Java SE 17 VMs. As such, the Eclipse SDK as a whole is targeted at all modern, desktop Java VMs.</p>
   <p>
-    <a href="#appendix">Appendix 1</a> contains a table that indicates the class library level required for each bundle.
+    In general, the 4.32 release of the Eclipse Project is developed on Java SE 17 VMs. As such, the Eclipse SDK as a whole is targeted at all modern, desktop Java VMs.
+    The IDE itself, however, requires Java 21 to run correctly, see <a href="https://www.eclipse.org/setups/sponsor/?scope=Eclipse%20IDE&campaign=2024-06">here</a> for more information.
```

Comment:
Change "IDE itself" with "EPP packages" and make it a link to https://www.eclipse.org/downloads/packages/.

Reply:
Done